rskj-core/src/main/java/co/rsk/net/BlockSyncService.java

```diff
@@ -102,7 +106,8 @@ public BlockProcessResult processBlock(MessageChannel sender, @Nonnull Block blo
 
         lastKnownBlockNumber = Math.max(blockNumber, lastKnownBlockNumber);
 
-        if (ignoreAdvancedBlocks && blockNumber >= bestBlockNumber + 1000) {
+        int syncMaxDistance = syncConfiguration.getMaxSkeletonChunks() * syncConfiguration.getChunkSize();
+        if (ignoreAdvancedBlocks && blockNumber > bestBlockNumber + syncMaxDistance) {
```

Comment:
Do we still wanat this ?

Reply:
We should discuss to remove it in the next version, if we want to keep blocks farther than one skeleton away. By now I would keep it